js/app.js

```diff
@@ -13,6 +13,8 @@ var Tracker = function () {
 	this.photoArray = [];
 }
 
+var tracker = new Tracker();
```

Comment:
Just slacked it to you. I copied and pasted exactly what you posted, then did remove one comma after "Vintage" to see if that did anything, but it is the same.